src/chip/_chip.scss

```diff
@@ -0,0 +1,63 @@
+
+@import "../variables";
+
+.mdl-chip {
+    height: $chip-height;
+    font-family: $preferred_font;
+    line-height: $chip-height;
+    padding: 0 12px;
+    border-radius: $chip-height / 2;
+    background-color: $chip-bg-color;
+    display: inline-block;
+    color: $text-color-primary;
+    margin: 2px 0;
+    font-size: 0;
+    
+    &.mdl-chip--deletable {
```

Comment:
Let's try to keep modifiers at the bottom of declarations. I've learned in recent experimenting it helps with the specificity flow.